pkg/edit/prompt.go

```diff
@@ -126,6 +127,67 @@ func defaultStaleTransform(original ui.Text) ui.Text {
 	return ui.StyleText(original, ui.Inverse)
 }
 
+type rawToken struct {
```

Comment:
This type is declared in the `edit` package, where `rawToken` is not descriptive enough; `sgrToken` is better.